tox.ini

```diff
@@ -144,12 +145,14 @@ commands =
     sphinx-build -E -b linkcheck doc ./doc/_build/linkcheck
 
 [testenv:test-atlas]
-description = run atlas connection tests
 deps =
-     {[testenv:test]deps}
+    {[testenv:aws-secrets]deps}
+    {[testenv:test]deps}
 passenv = *
+allowlist_externals = bash
 commands =
-    python -m pytest -v {posargs} ./test/atlas/test_connection.py
+    {[testenv:aws-secrets]commands}
```

Comment:
I think `test-atlas` can go away altogether.  What I'm thinking is:

- Call `tox -m aws-secrets -- drivers/atlas_connect`
- Call `TEST_ATLAS=1 tox -m test-eg`
- In `run-tests.sh`, test if `./secrets-export.sh` exists and source it

Reply:
I like this idea a lot!

Reply:
Can you also remove `test-atlas` from tox altogether?

Reply:
Missed that, whoops!